packages/components/inputs/rich-text-input/src/rich-text-input.tsx

```diff
@@ -49,25 +21,11 @@ type TChangeEvent = {
   };
 };
 
-type TFocusEvent = TBaseEvent;
-
-type TEventHook<T = Event> = (
-  event: T,
-  editor: typeof Editor,
-  next: () => unknown
-) => unknown;
-
-type TOnChangeParam = {
-  operations: unknown;
-  value: ReturnType<typeof html.deserialize>;
-};
-type TOnChangeFn = (change: TOnChangeParam) => unknown;
-
 export type TRichTextInputProps = {
   /**
    * Focus the control when it is mounted
    */
-  isAutofocussed?: boolean;
+  isAutofocused?: TEditorProps['isAutofocused'];
```

Comment:
It seems this would also be a breaking change since all consumers using the old property name would see their behaviour modified.

Reply:
good point, I will mention that in the changeset